docs/references/api/resource_embedding.rst

```diff
@@ -821,61 +821,31 @@ Response:
    }
 
 .. _embed_disamb:
-
-Embedding Disambiguation
-------------------------
-
-For doing resource embedding, PostgREST infers the relationship between two tables based on a foreign key between them.
-However, in cases where there's more than one foreign key between two tables, it's not possible to infer the relationship unambiguously
-by just specifying the tables names.
-
+.. _hint_disamb:
 .. _target_disamb:
+.. _multiple_relationships:
 
-Target Disambiguation
-~~~~~~~~~~~~~~~~~~~~~
+Multiple Relationships
+----------------------
 
-For example, suppose you have the following ``orders`` and ``addresses`` tables:
+When there are multiple foreign keys between two or more tables, PostgREST does not infer the embedding directly.
+You need to use :ref:`computed_relationships` in these cases. For example, suppose you have the following ``orders`` and ``addresses`` tables:
 
 .. image:: ../../_static/orders.png
 
-And you try to embed ``orders`` with ``addresses`` (this is the **target**):
-
-.. tabs::
-
-  .. code-tab:: http
-
-    GET /orders?select=*,addresses(*) HTTP/1.1
-
-  .. code-tab:: bash Curl
-
-    curl "http://localhost:3000/orders?select=*,addresses(*)" -i
-
-Since the ``orders`` table has two foreign keys to the ``addresses`` table — an order has a billing address and a shipping address —
-the request is ambiguous and PostgREST will respond with an error:
-
-.. code-block:: http
-
-   HTTP/1.1 300 Multiple Choices
-
-   {..}
-
-If this happens, you need to disambiguate the request by adding precision to the **target**.
-Instead of the **table name**, you can specify the **foreign key constraint name** or the **column name** that is part of the foreign key.
-
-Let's try first with the **foreign key constraint name**. To make it clearer we can name it:
+To successfully embed ``orders`` with ``addresses``, you need to create computed relationships for the foreign keys that you want to use:
```

Comment:
Started directly with the solution without mentioning the `300 Multiple Choices`  error as mentioned in point 2 of https://github.com/PostgREST/postgrest/issues/2863.